modules/cas_cache/layer_upgrade.c

```diff
@@ -305,6 +314,66 @@ static int _cas_upgrade_dump_cache_conf_cores(ocf_cache_t device,
 	return result;
 }
 
+static int _cas_upgrade_dump_cache_conf_promotion(ocf_cache_t cache,
+	struct cas_properties *cache_props)
+{
+	uint32_t promotion_type;
```

Comment:
I would initialize this to some default to avoid comparing of potentially uninitialized variable in line 327

Reply:
Done